crates/router/src/core/payouts/transformers.rs

```diff
@@ -103,3 +110,78 @@ impl
         }
     }
 }
+
+impl
+    ForeignFrom<(
+        &PayoutRequiredFields,
+        Vec<EnabledPaymentMethod>,
+        api::RequiredFieldsOverrideRequest,
+    )> for Vec<api::PayoutEnabledPaymentMethodsInfo>
+{
+    fn foreign_from(
+        (payout_required_fields, enabled_payout_methods, value_overrides): (
+            &PayoutRequiredFields,
+            Vec<EnabledPaymentMethod>,
+            api::RequiredFieldsOverrideRequest,
+        ),
+    ) -> Self {
+        let value_overrides = value_overrides.flatten_struct_keys();
+
+        enabled_payout_methods
+            .into_iter()
+            .map(|enabled_payout_method| {
+                let payment_method = enabled_payout_method.payment_method;
+                let payment_method_types_info = enabled_payout_method
+                    .payment_method_types
+                    .into_iter()
+                    .filter_map(|pmt| {
+                        payout_required_fields
+                            .0
+                            .get(&payment_method)
+                            .and_then(|pmt_info| {
+                                pmt_info.0.get(&pmt).map(|connector_fields| {
+                                    let mut required_fields = IndexMap::new();
+
+                                    for required_field_final in connector_fields.fields.values() {
+                                        required_fields.extend(required_field_final.common.clone());
+                                        required_fields
+                                            .extend(required_field_final.mandate.clone());
+                                        required_fields
+                                            .extend(required_field_final.non_mandate.clone());
```

Comment:
is this expected to extend required_fields with both mandate and non_mandate fields without any validation?

Reply:
There is no mandate concept in payouts, so we can completely remove the extension to `mandate` and `non_mandate` fields